File: Selenium/Activity12.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support import expected_conditions as EC
from selenium.webdriver.support.wait import WebDriverWait

with webdriver.Chrome() as driver:
    wait = WebDriverWait(driver, timeout=10)
    driver.get("https://training-support.net/webelements/dynamic-content")
    print(driver.title)
    button = driver.find_element(By.ID, "genButton")
    button.click()
    if(wait.until(EC.text_to_be_present_in_element((By.ID, "word"), "release"))):
        print(driver.find_element(By.ID,"word").text)
    #driver.quit()

File: Selenium/Activity12.java
package Selenium;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity12 {

	public static void main(String[] args) {
		WebDriver driver = new ChromeDriver();
		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(60));
		driver.manage().window().maximize();
		driver.get("https://training-support.net/webelements/dynamic-content");
		System.out.println("Page Title is : " + driver.getTitle());
		WebElement button = driver.findElement(By.id("genButton"));
		button.click();
		if(wait.until(ExpectedConditions.textToBePresentInElementLocated(By.id("word"),"release"))) {
			System.out.println("Founded word is : " + driver.findElement(By.id("word")).getText());
		}
		driver.quit();
	}

}
